docs/sources/features/datasources/elasticsearch.md

```diff
@@ -91,6 +91,15 @@ For example, if you're using a default setup of Filebeat for shipping logs to El
 - **Message field name:**  message
 - **Level field name:** fields.level
 
+### Data links
+
+Data links create a link from specified field that can be accessed in logs view in Explore.
+
+Each data link configuration consists of: 
+- **Field**: Name of the field used by the data link
+- **Url/Query**: If link is external, you fill in full url for the link. In case of internal link this input serves as query for the target data source. In both cases you can interpolate the value from the field with `${__value.raw }` macro.
+- **Internal link**: Switch to select if link is internal or external. In case of internal link, data source selector will be shown where you can select the target data source. At this moment only tracing data sources are supported.
```

Comment:
```suggestion
- **Internal link -** Select if the link is internal or external. In case of internal link, a data source selectorallows you to select the target data source. Only tracing data sources are supported.
```